docs/.gitignore

```diff
@@ -1,21 +1,35 @@
-.hugo_build.lock
-assets/doc-models/*.json
-bin/
-content/docs/api
-node_modules/
-public/
-scripts/api-documenter/temp
-themes/thxvscode/assets/js/node_modules/
+# TODO: can probably be removed once we move to pipeline-based deployment
+.env
```

Comment:
This PR moves to pipeline based deployment, right?

Reply:
It does, though this comment is out of date. We were using this for Azure credentials when publishing to the test environment from our local dev environment. But now we're using `.env` for build variables, so this should remain git-ignored. I'll remove the comment. Good catch!